docs/semgrep-code/pro-rules.md

```diff
@@ -8,95 +8,67 @@ tags:
 
 :::info
 Rules that don't apply to your target repository's language or framework are skipped automatically even if they are in your Policies page. For example, if your repository contains JavaScript code and you have added Go rules, the Go rules are unused. Unused rules do not add to scan time.
 :::
 
-### Adding Semgrep Pro rules in CLI or CI
-
-:::note Prerequisites
-For CLI users: You must be [logged in](/getting-started/cli#log-in-to-your-semgrep-account).
-
-:::
-
-1. Go to [Semgrep Registry](https://semgrep.dev/r).
-2. Click on **Visibility > Pro rules**.
-3. Optional: Apply additional filters by entering search terms in the search box or selecting filters from drop-down boxes.
-4. For a single rule, click the rule's card > Run locally. For rulesets, click the card.
-5. Copy and paste the snippet to your CLI or CI configuration file. You can add several rulesets. Refer to the following sample snippet:
+### Change rule modes or disable Pro rules in Semgrep AppSec Platform
 
-### Adding Pro rules to your Semgrep org through the Registry
+Like any other rule or ruleset, you can disable Pro rules or change their rule mode to leave comments to developers or potentially block a PR.
 
 1. Sign in to [Semgrep AppSec Platform](https://semgrep.dev/login).
-2. Go to [Semgrep Registry](https://semgrep.dev/r).
-3. Click on **Visibility > Pro rules**.
-4. Optional: Apply additional filters by entering search terms in the search box or selecting filters from drop-down boxes.
-5. To add a specific rule or ruleset:
-    1. Click on the card for the **rule or ruleset > Add to Policy**.
-    2. Select a Policy rule mode to add the rules to. It is recommended to start with the Monitor mode to silently gather findings and audit the rule's performance.
-
-### Filtering behavior
-
-* Filter types such as Language and Technology use AND logic. This means that search terms must match all filters. For example, selecting Java (a Language) and security (a Category) shows only rules with both properties (Java and security).
-* Adding filters of the same type use OR logic. This means that search terms can match any of the filters for that type. For example, selecting Java and Python (both languages) shows rules with either language.
-* A gem icon (💎) denotes Semgrep Pro rules.
-
-<!-- Future feature: Add Semgrep Pro rules through your Policy -->
+1. Navigate to **Rules > Policies**.
+1. Under **Source**, click **Pro <i class="fa-solid fa-gem"></i>** to view all the Semgrep Pro rules currently in your Policies.
+1. Find and select the rules you want to disable or change.
+1. Click **Change modes** and select one of the provided options.
 
-<!-- Future feature: Receiving updates on Semgrep Pro rules -->
-
-## Removing or disabling Semgrep Pro rules
+You can find all Semgrep Pro rules in your Policies page, so if you want to re-enable Pro rules, you can simply use the **Source > Pro <i class="fa-solid fa-gem"></i>** filter.

```

Comment:
```suggestion
You can find all previously-added Semgrep Pro rules in your Policies page, so if you want to re-enable Pro rules or adjust the mode again in the future, use the **Source > Pro <i class="fa-solid fa-gem"></i>** filter as described previously.
```
exact wording here I'm not picky about, but saying "all" seems confusing to me, as it's not all, just all the ones you had originally